src/content/en/2019/compression.md

```diff
@@ -213,32 +220,33 @@ Additionally, the percentage of Brotli compression is higher for third party con
    <td><p style="text-align: right">0.01%</p></td>
   </tr>
   <tr>
-   <td>other / invalid
-   </td>
+   <td><em>Other / Invalid</em></td>
    <td><p style="text-align: right">0.01%</p></td>
    <td><p style="text-align: right">0.02%</p></td>
    <td><p style="text-align: right">0.01%</p></td>
    <td><p style="text-align: right">0.02%</p></td>
   </tr>
 </table>
 
-# Identifying Compression Opportunities
+## Identifying Compression Opportunities
 
-[Google’s Lighthouse](https://developers.google.com/web/tools/lighthouse) tool enables users to run a series of audits against web pages, and[ one of them](https://developers.google.com/web/tools/lighthouse/audits/text-compression) evaluates whether a site can benefit from additional text based compression. It does this by attempting to compress resources and evaluate whether an object’s size can be reduced by at least 10% and 1400 bytes. Depending on the score, you may see a compression recommendation in the results, with a list of specific resources that could be compressed.
+[Google’s Lighthouse](https://developers.google.com/web/tools/lighthouse) tool enables users to run a series of audits against web pages, and [one of them](https://developers.google.com/web/tools/lighthouse/audits/text-compression) evaluates whether a site can benefit from additional text based compression. It does this by attempting to compress resources and evaluate whether an object’s size can be reduced by at least 10% and 1400 bytes. Depending on the score, you may see a compression recommendation in the results, with a list of specific resources that could be compressed.
```

Comment:
Linkifying only the tool name
Rephrased the sentence linking to the compression audit
Hyphenate "text-based"
Use a comma to separate thousands place in large numbers ("1,400" as opposed to "1400")
```suggestion
Google’s [Lighthouse](https://developers.google.com/web/tools/lighthouse) tool enables users to run a series of audits against web pages. The [text compression audit](https://developers.google.com/web/tools/lighthouse/audits/text-compression) evaluates whether a site can benefit from additional text-based compression. It does this by attempting to compress resources and evaluate whether an object’s size can be reduced by at least 10% and 1,400 bytes. Depending on the score, you may see a compression recommendation in the results, with a list of specific resources that could be compressed.
```